src/components/Cell.tsx

```diff
@@ -16,6 +17,8 @@ import {
 import { ThemeContext } from './Theme';
 
 export interface CellInterface {
+  accessibilityHint?: string;
+  accessibilityRole?: AccessibilityRole;
```

Comment:
Update docs to reflect the newly introduced props: `accessibilityHint`, `accessibilityRole`, `accessibilityLabel`, `accessibilityState`